ui/js/actions/app.js

```diff
@@ -84,6 +84,15 @@ export function doHistoryBack() {
   };
 }
 
+export function doHistoryForward() {
+  return function(dispatch, getState) {
+    //if (!history.state) return;
+    //if (history.state.index === 0) return;
+
```

Comment:
Well you could comment out the same statements in the "back" function and check what breaks(most probably clicking back when the history is empty), and then try to replicate the similar problem with the "forward" button. Although this statement `if (!history.state) return;` seems kind of necessary, the next one, not so much...